packages/manager/modules/pci/src/projects/project/kubernetes/kubernetes.routing.js

```diff
@@ -42,6 +43,13 @@ export default /* @ngInject */ ($stateProvider) => {
           OvhApiCloud => OvhApiCloud.v6().schema().$promise
             .then(schema => get(schema, VERSION_ENUM_KEY)),
 
+        highestVersion: /* @ngInject */
+          versions => reduce(versions, (currentHighestVersion, version) => (
+            parseFloat(version) > parseFloat(currentHighestVersion)
```

Comment:
```suggestion
            parseFloat(`${version}`) > parseFloat(`${currentHighestVersion}`)
```

prevents a bug with octal and `parseFloat` :)

Reply:
This is fixed. Thanks!